opencga-server/src/main/java/org/opencb/opencga/server/rest/MetaWSServer.java

```diff
@@ -98,6 +99,19 @@ public Response ping() {
         return createOkResponse(queryResult);
     }
 
+    @GET
+    @Path("/model")
+    @ApiOperation(httpMethod = "GET", value = "Opencga model webservices.", response = String.class)
+    public Response model(@QueryParam("model") String modelStr) {
+        OpenCGAResult<String> queryResult = null;
```

Comment:
We should start using run method to avoid boilerplate try-catch code in the webservices. Lines 106-112 would result in a single line:
`return run(() -> new OpenCGAResult<>(0, Collections.emptyList(), 1, Collections.singletonList(DataModelsUtils.dataModelToJsonString(Class.forName(modelStr), false)), 1));`